pacts/cli-lobby.json

```diff
@@ -0,0 +1,48 @@
+{
+  "consumer": {
+    "name": "CLI"
+  },
+  "provider": {
+    "name": "Lobby"
+  },
+  "interactions": [
+    {
+      "description": "A request to create a lobby",
+      "providerState": "The server is online",
+      "request": {
+        "method": "POST",
+        "path": "/lobby",
+        "headers": {
+          "Content-Type": "application/json"
+        }
+      },
+      "response": {
```

Comment:
I understand here begins the expected response.

Reply:
indeed!